packages/debian/entrypoint.sh

```diff
@@ -32,20 +32,53 @@ rpm2deb() {
     pushd /debbuild/result 1> /dev/null
     alien --bump=0 /rpmbuild/source.rpm
 
-    chown "${TARGET_UID}:${TARGET_GID}" * 
+    chown "${TARGET_UID}:${TARGET_GID}" *
 }
 
-case ${1:- } in
-    builddeb)
-        builddeb
-        ;;
-    rpm2deb)
-        rpm2deb
-        ;;
-    '')
-        exec /bin/bash
-        ;;
-    *)
-        exec "$@"
-        ;;
-esac
+
+download_packages() {
+
+    local -a list_pkg("$@")
+
+    # Salt-minion GPG Key
+    curl -s https://repo.saltstack.com/apt/ubuntu/18.04/amd64/2018.3/SALTSTACK-GPG-KEY.pub | apt-key add -
+    echo 'deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/2018.3 bionic main' >> /etc/apt/sources.list.d/saltstack.list
+
+    # Kubectl & Kubelet GPG key
+    echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" >>/etc/apt/sources.list.d/kubernetes.list
+    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+
+    mkdir /packages/pkg_downloaded
+    for pkg in $(list_pkg)
+    do
+        echo "$pkg" >> packages_list.txt
+        for dep in $(apt-rdepends "$pkg" | grep -E 'Depends' | cut -d " " -f4)
+            do echo "$dep" >> packages_list.txt
+        done
+    done
+
+    pushd /packages/pkg_downloaded
+    for pkg2dl in $(sort -u /packages_list.txt)
+    #Do not copy this way to itterate packages, it's work because there is no space
```

Comment:
```suggestion
    #Do not copy this way to iterate on packages, it's working because there is no space in the names
```

Reply:
Are we sure this is a very reliable way of doing it? Because as you are saying, if we had a name with a space, it could break